src/t_cose_util.h

```diff
@@ -105,6 +126,33 @@ enum t_cose_err_t create_tbs_hash(int32_t                     cose_algorithm_id,
                                   struct q_useful_buf         buffer_for_hash,
                                   struct q_useful_buf_c      *hash);
 
+/**
+ * Serialize the to-be-signed (TBS) bytes for COSE.
+ *
+ * \param[in] protected_parameters  Full, CBOR encoded, protected parameters.
+ * \param[in] aad                   Additional Authenitcated Data to be
+ *                                  included in TBS.
+ * \param[in] payload               The CBOR-encoded payload.
+ * \param[in] buffer_for_hash       Pointer and length of buffer into which
+ *                                  the resulting TBS bytes is put.
+ * \param[out] hash                 Pointer and length of the
+ *                                  resulting TBS bytes.
```

Comment:
buffer_for_hash and hash do not correspond to the actual variable names.

Reply:
Done.